.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": true,
```

Comment:
I added it here to make it **clear** that semicolons **should** be used in this project and to let others know that it's not because we're using Prettier's default settings, but because it's **necessary**.